commands/deploy.go

```diff
@@ -464,7 +464,7 @@ func getRegistryAuth(config *configFile, image string) string {
 		slashes := strings.Count(image, "/")
 		if slashes > 1 {
 			regS := strings.Split(image, "/")
-			registry = strings.Join(regS[:len(regS)-2], ", ")
+			registry = regS[0]
 		}
```

Comment:
Do we need to check this index first?

Reply:
@johnmccabe

Reply:
Nope, since you've checked that the number of slashes is > 0 above

Reply:
ty